packages/components/src/components/button/button.utils.ts

```diff
@@ -0,0 +1,26 @@
+import { ICON_BUTTON_SIZES, PILL_BUTTON_SIZES } from './button.constants';
+
+const isValidIconSize = (
+  size: number,
+  variant: string,
+): number | boolean => Object.values(ICON_BUTTON_SIZES).includes(size)
+    && !(size === 20 && variant !== 'tertiary');
```

Comment:
```suggestion
    && !(size === 20 && variant !== BUTTON_VARIANTS.TERTIARY);
```
Always use constants instead of strings, whenever wherever possible.